packages/@guardian/source-react-components-development-kitchen/src/toggle-switch/ToggleSwitch.tsx

```diff
@@ -82,12 +88,14 @@ export const ToggleSwitch = ({
 	defaultChecked,
 	cssOverrides,
 	platform = 'web',
+	disabled = false,
 	onClick = () => undefined,
-	...props
```

Comment:
Is there a reason to delete this?

Reply:
Yeah, because I deleted the bottom one, then this was showing up as never being used. I will add that back in.